main/README.md

```diff
@@ -4,53 +4,109 @@
 # run `yarn docs:dev` at any time to start local dev server and access
 # website at localhost:8080 by default
 ####
-home: true # use default home page layout (hero image with text, features section)
-heroImage: /logo.svg
-## Action button
-actionText: Get Started → # text that goes in the button
-actionLink: /guides/getting-started/ # go-to link when clicking on button
-
-## Features section
-features:
-  - title: Electronic Rights Transfer Protocol (ERTP)
-    details: Agoric empowers individuals to securely execute transactions, establish new markets, and craft novel patterns of exchange — without centralized control.
-
-  - title: Better security architecture with OCaps
-    details: Agoric uses an object-capability model (OCaps) security architecture, in which access to a programming object itself is the authority to use the object.
-
-  - title: Securely Create and Transfer Assets
-    details: All kinds of digital assets can be easily created, but importantly, they can be transferred in exactly the same ways, with exactly the same security properties.
-
-footer: Apache-2.0 Licensed | Copyright © 2022 - Agoric
+home: false # use default home page layout (hero image with text, features section)
+footer: Apache-2.0 Licensed | Copyright © 2023 - Agoric
 ---
 
-<div class="flex flex--column flex--center">
-  <p>
-    Learn about ERTP (Electronic Rights Transfer Protocol), a uniform way of transferring tokens and other digital assets in JavaScript.
-  </p>
-  <Button-Action-Link
-    text="Explore ERTP"
-    link="/guides/ertp/"
-  />
+<div class="home-banner">
+  Receive our latest tutorials, webinar invites, and much more! <a href="https://agoric.com/dev-newsletter" style="color: white;"">Join our newsletter</a>
 </div>
-<br>
-<div class="flex flex--column flex--center">
-  <p>Check out Zoe, a smart contract framework responsible for enforcing what we call "offer safety". The smart contracts that run on top of Zoe are responsible for figuring out proposed reallocations of resources.
-  </p>
-  <Button-Action-Link
-    text="Build on Zoe"
-    link="/guides/zoe/"
+
+<div class="home-section">
+  <HomeButtonHeader
+    title="Agoric documentation"
+    text="The Agoric platform makes it possible to write safer smart contracts with your JavaScript skillset."
+  />
+  <HomeButtonRow
+    title1="What's Agoric"
+    text1="Learn about Agoric and its Javascript platform"
+    link1="/guides/what-is-agoric/"
+    title2="Getting Started"
+    text2="Set up your environment and start building apps"
+    link2="/guides/getting-started/"
+    title3="Contract Framework"
+    text3="Set up your environment and start building"
+    link3="/guides/zoe/"
   />
+  <HomeButtonRow
+    title1="Token Standards"
+    text1="Understand the ins and outs of Agoric's ERTP standard"
```

Comment:
"ERTP standard" seems a little like ATM machine. ERTP is a protocol. Electronic Rights Transfer Protocol.

```suggestion
    title1="Token Protocol"
    text1="Understand the ins and outs of Agoric's Electronic Rights Transfer Protocol (ERTP)"
```